scripts/solid-js/README.md

```diff
@@ -83,7 +83,7 @@ Create (or modify) a document in your Fireproof database.
 
 ```ts
 // API Signature
-function createDocument<T extends DocRecord<T>>(initialDocFn: Accessor<Doc<T>>): CreateDocumentResult<T>;
+function createDocument<T extends DocTypes>(initialDocFn: Accessor<Doc<T>>): CreateDocumentResult<T>;
```

Comment:
we should do a pass before merge to look at all the public changes and consider which ones make sense to un-churn. I think this is better but I wanna see how they stack up in aggregate. 

Maybe we have the improvements for the next release to make eval easier.